README.md

```diff
@@ -166,6 +166,38 @@ receivers:
   - service_key: <team-DB-key>
 ```
 
+## High Availability
+
+> Warning: High Availablility is under active development
+
+To create a highly available cluster of the Alertmanager the instances need to
+be configured to communicate with each other. This is configured using the
+`-mesh.*` flags.
+
+- `-mesh.hardware-address` string: MAC address, i.e. mesh peer ID (default "&lt;hardware-mac-address&gt;")
+- `-mesh.listen-address` string: mesh listen address (default "0.0.0.0:6783")
+- `-mesh.nickname` string: peer nickname (default "&lt;machine-hostname&gt;")
+- `-mesh.peer` value: initial peers (may be repeated)
+
+The `mesh.hardware-address` flag is used as a unique ID among the peers. It
+defaults to the mac address, therefore the default value should typically be a
+good option. The same applies to the default of the `mesh.nickname` flag, as it
+defaults to the hostname. The chosen port in the `mesh.listen-address` flag is
+the port that needs to be specified in the `mesh.peer` flag of the other peers.
```

Comment:
Thinking again, that's probably not necessarily true for prod setups. Guess it's fine for testing and people will get the drift if they have their own port mapping for containers etc.